### PR TITLE
[DOC-2145] update docs

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Trying to organize docs in the following way:
Readme in the root: https://github.com/datarobot/datarobot-user-models contains information about every aspect of custom models:  Quickstart, examples, how to assemble folder, etc
Readme in the: https://github.com/datarobot/datarobot-user-models/custom_model_runner - contains everything about drum and references to the first document.

DR App docs will be changed in the way to refer to two of these readmes

### CODE REVIEW

**Comment by yakov-g** [comment]
jarvis test functional_drop_in

File: README.md [hunk 1]
```diff
@@ -1,32 +1,144 @@
...
+5. [Custom Environment Templates](#custom_environment_templates)
+6. [Custom Model Runner (drum)](#custom_model_runner)
+4. [Contribution & development](#contribution_development)
+
+## What is this repository? <a name="what_is_it"></a>
+The **DataRobot User Models** repository contains information and tools for assembling,
+debugging, testing, and running your training and inference models with DataRobot.
+
+#### Terminology
+This repository address the DataRobot functionality known as `custom models`. The terms `custom model` and `user model` can be used interchangeably, as can `custom model directory` and `code directory`.
```
Comment (4dahalibut) [inline README.md:18]: I thought the terms were different... a custom model directory is a code directory with a model in it.

Comment (4dahalibut) [reply README.md:18]: is custom model directory still used?

File: README.md [hunk 2]
```diff
@@ -1,32 +1,144 @@
...
+
+## Quickstart <a name="quickstart"></a>
+The following example shows how to use the [**drum**](https://github.com/datarobot/datarobot-user-models/tree/master/custom_model_runner) tool to make predictions on an [sklearn model](model_templates/inference/python3_sklearn)
+1. Clone the repository
+2. Create a virtual environment: `python3 -m virtualenv <dirname for virtual environment>`
+3. Activate the virtual environment: `source <dirname for virtual environment>/bin/activate`
+4. cd into the repo: `cd datarobot-user-models`
+5. Install the required dependencies: `pip install -r public_dropin_environments/python3_sklearn/requirements.txt`
+6. Install datarobot-drum: `pip install datarobot-drum`
```
Comment (4dahalibut) [inline README.md:27]: all of this could be removed with my plan!

File: README.md (hunk 1 above)
Comment (yakov-g) [reply README.md:18]: yes, it is used.
If we want to unify, let's make it `code dir` everywhere: in training and in inference.
But people will continue to calling it `custom model dir`, so better to have this disclaimer.

File: README.md [hunk 3]
```diff
@@ -1,32 +1,144 @@
...
+
+### Built-In Model Support
+The **drum** tool has built-in support for the following libraries. If your model is based on one of these libraries, **drum** expects your model artifact to have a matching file extension.
+
+### Python libraries
+| Library | File Extension | Example |
+| --- | --- | --- |
+| scikit-learn | *.pkl | sklean-regressor.pkl |
+| xgboost | *.pkl | xgboost-regressor.pkl |
+| PyTorch | *.pth | torch-regressor.pth |
```
Comment (4dahalibut) [inline README.md:50]: pytorch and java don't work for training models yet...

Comment (yakov-g) [reply README.md:50]: this is for inference only

Comment (yakov-g) [reply README.md:50]: I'll update section header

File: README.md (hunk 2 above)
Comment (yakov-g) [reply README.md:27]: What is your plan?

Comment (4dahalibut) [reply README.md:27]: to build a docker image locally and run drum in it

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**6.42** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**